<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>套打助手</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>模板管理</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnModType">
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="tabletTracking">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>管理模板类别</string>
               </property>
               <property name="text">
                <string>类别管理</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnModLoad">
               <property name="toolTip">
                <string>导入外部模板</string>
               </property>
               <property name="text">
                <string>导入模板</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnModExp">
               <property name="toolTip">
                <string>将当前模板导出为独立模板文件</string>
               </property>
               <property name="text">
                <string>导出模板</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnModNew">
               <property name="toolTip">
                <string>新建模板</string>
               </property>
               <property name="text">
                <string>新建</string>
               </property>
               <property name="icon">
                <iconset resource="printres.qrc">
                 <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnModModi">
               <property name="toolTip">
                <string>更改当前模板</string>
               </property>
               <property name="text">
                <string>更改</string>
               </property>
               <property name="icon">
                <iconset resource="printres.qrc">
                 <normaloff>:/res/edit.png</normaloff>:/res/edit.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnModDel">
               <property name="toolTip">
                <string>删除当前模板</string>
               </property>
               <property name="text">
                <string>删除</string>
               </property>
               <property name="icon">
                <iconset resource="printres.qrc">
                 <normaloff>:/res/del.png</normaloff>:/res/del.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>查找：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditQuery"/>
             </item>
             <item>
              <widget class="QToolButton" name="btnQuery">
               <property name="text">
                <string>查</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>res/query.png</normaloff>res/query.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QToolButton" name="btnPrintSingle">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolTip">
              <string>打印当前票据</string>
             </property>
             <property name="text">
              <string>单张打印</string>
             </property>
             <property name="icon">
              <iconset resource="printres.qrc">
               <normaloff>:/res/print.png</normaloff>:/res/print.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnPrintNewData">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolTip">
              <string>清空当前票据中的数据，重新录入</string>
             </property>
             <property name="text">
              <string>重新录入</string>
             </property>
             <property name="icon">
              <iconset resource="printres.qrc">
               <normaloff>:/res/new.png</normaloff>:/res/new.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnPrintSaveData">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolTip">
              <string>保存当前票据到票据仓库中</string>
             </property>
             <property name="text">
              <string>保存票据</string>
             </property>
             <property name="icon">
              <iconset resource="printres.qrc">
               <normaloff>:/res/save.png</normaloff>:/res/save.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnPrintDatas">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolTip">
              <string>从票据仓库中读入到当前票据</string>
             </property>
             <property name="text">
              <string>票据仓库</string>
             </property>
             <property name="icon">
              <iconset resource="printres.qrc">
               <normaloff>:/res/custom.png</normaloff>:/res/custom.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnBatchPrint">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolTip">
              <string>根据批量数据文件打印多张凭证</string>
             </property>
             <property name="text">
              <string>批量打印</string>
             </property>
             <property name="icon">
              <iconset resource="printres.qrc">
               <normaloff>:/res/copy.png</normaloff>:/res/copy.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnPrintSetup">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="toolTip">
              <string>设置打印机参数</string>
             </property>
             <property name="text">
              <string>打印设置</string>
             </property>
             <property name="icon">
              <iconset resource="printres.qrc">
               <normaloff>:/res/setup.png</normaloff>:/res/setup.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextUnderIcon</enum>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>520</width>
             <height>441</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>模板管理</string>
     </property>
     <property name="icon">
      <iconset resource="printres.qrc">
       <normaloff>:/res/home.png</normaloff>:/res/home.png</iconset>
     </property>
     <addaction name="actionmenuType"/>
     <addaction name="separator"/>
     <addaction name="actionmenuModNew"/>
     <addaction name="actionmenuModModify"/>
     <addaction name="actionmenuModDelete"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="actionprintdatas"/>
    <addaction name="separator"/>
    <addaction name="actionmodload"/>
    <addaction name="actionmodexport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionprintsetup"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="separator"/>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
  </widget>
  <action name="action_3">
   <property name="text">
    <string>打印机设置</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/exit.png</normaloff>:/res/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionsysbackup">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/approve.png</normaloff>:/res/approve.png</iconset>
   </property>
   <property name="text">
    <string>系统备份</string>
   </property>
  </action>
  <action name="actionsysrestore">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/convert.png</normaloff>:/res/convert.png</iconset>
   </property>
   <property name="text">
    <string>系统恢复</string>
   </property>
  </action>
  <action name="actionprintsetup">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/setup.png</normaloff>:/res/setup.png</iconset>
   </property>
   <property name="text">
    <string>打印设置</string>
   </property>
  </action>
  <action name="actionsystemsetup">
   <property name="icon">
    <iconset>
     <normaloff>res/setup.png</normaloff>res/setup.png</iconset>
   </property>
   <property name="text">
    <string>系统参数</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/option.png</normaloff>:/res/option.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/info.png</normaloff>:/res/info.png</iconset>
   </property>
   <property name="text">
    <string>关于 ...</string>
   </property>
  </action>
  <action name="actionmodload">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/preview.png</normaloff>:/res/preview.png</iconset>
   </property>
   <property name="text">
    <string>导入模板</string>
   </property>
  </action>
  <action name="actionmodexport">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/forward.png</normaloff>:/res/forward.png</iconset>
   </property>
   <property name="text">
    <string>导出模板</string>
   </property>
  </action>
  <action name="actiongenggai">
   <property name="text">
    <string>更改模板</string>
   </property>
  </action>
  <action name="actionshanchu">
   <property name="text">
    <string>shanchu </string>
   </property>
  </action>
  <action name="actionprintdatas">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/custom.png</normaloff>:/res/custom.png</iconset>
   </property>
   <property name="text">
    <string>票据仓库</string>
   </property>
  </action>
  <action name="actioninfodatas">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/edit.png</normaloff>:/res/edit.png</iconset>
   </property>
   <property name="text">
    <string>备选信息</string>
   </property>
  </action>
  <action name="actionmenuType">
   <property name="icon">
    <iconset>
     <normaloff>res/document.png</normaloff>res/document.png</iconset>
   </property>
   <property name="text">
    <string>类别管理</string>
   </property>
  </action>
  <action name="actionmenuTypeRename">
   <property name="text">
    <string>类别重命名</string>
   </property>
  </action>
  <action name="actionmenuTypeDelete">
   <property name="text">
    <string>删除类别</string>
   </property>
  </action>
  <action name="actionmenuModNew">
   <property name="icon">
    <iconset>
     <normaloff>res/add.png</normaloff>res/add.png</iconset>
   </property>
   <property name="text">
    <string>新建模板</string>
   </property>
  </action>
  <action name="actionmenuModModify">
   <property name="icon">
    <iconset>
     <normaloff>res/edit.png</normaloff>res/edit.png</iconset>
   </property>
   <property name="text">
    <string>更改模板</string>
   </property>
  </action>
  <action name="actionmenuModDelete">
   <property name="icon">
    <iconset>
     <normaloff>res/del.png</normaloff>res/del.png</iconset>
   </property>
   <property name="text">
    <string>删除模板</string>
   </property>
  </action>
  <action name="actionsyssetup">
   <property name="icon">
    <iconset resource="printres.qrc">
     <normaloff>:/res/address.png</normaloff>:/res/address.png</iconset>
   </property>
   <property name="text">
    <string>系统参数</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="printres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
